index.js

```diff
@@ -69,6 +69,15 @@ function applyStyle() {
 	var nestedStyles = this._styles;
 	var i = nestedStyles.length;
 
+	// Turns out that on Windows dimmed gray text becomes invisible in cmd.exe,
+	// see https://github.com/chalk/chalk/issues/58
+	// If we're on Windows and we're dealing with a gray color, temporarily make 'dim' a noop.
+	var originalDim = ansiStyles.dim.open;
+	if ((nestedStyles.indexOf('gray') != -1 || nestedStyles.indexOf('grey') != -1) &&
+			process.platform === 'win32' && !/^xterm/i.test(process.env.TERM)) {
+		ansiStyles.dim.open = '';
```

Comment:
Considered that. Don't really think it matters, decided to go for the simplest solution.